x-pack/platform/plugins/shared/streams/server/routes/content/route.ts

```diff
@@ -50,9 +51,7 @@ const exportContentRoute = createServerRoute({
   }),
   security: {
     authz: {
-      enabled: false,
-      reason:
-        'This API delegates security to the currently logged in user and their Elasticsearch permissions.',
+      requiredPrivileges: [STREAMS_FEATURE_PRIVILEGES.assets, STREAMS_FEATURE_PRIVILEGES.read],
```

Comment:
This means assets _and_ read are both required, which can only happen for the "all" access privilege, therefore just checking for assets is enough

Reply:
Alternatively, if you want to be able to access this route with either privilege you can use
`requiredPrivileges: [{ anyRequired: ['<privilege_1>', '<privilege_2>'] }],`

Reply:
simplified to just read